libraries/Microsoft.Bot.Configuration/BotConfiguration.cs

```diff
@@ -84,11 +84,22 @@ public static async Task<BotConfiguration> LoadFromFolderAsync(string folder, st
         }
 
         /// <summary>
-        /// load the configuration from a .bot file.
+        /// Load the bot configuration by looking in a folder and ing the first .bot file in the folder.
```

Comment:
"and ing" is a typo. Not sure what text is intended.

Reply:
Lol this is in many places in the class, fixing them all.